app/helpers/application_helper.rb

```diff
@@ -1381,4 +1381,8 @@ def remote_function(options)
   end
 
   attr_reader :big_iframe
+
+  def appliance_name
+    MiqServer.my_server.settings.fetch_path(:server, :appliance_name)
```

Comment:
There should be no difference in the value. The server name is pulled out of the config, from the same place, and saved in MiqServer#name at startup. I would recommend  MiqServer.my_server.name as well.